drivers/storage/portworx/component/monitoring.go

```diff
@@ -195,6 +436,13 @@ func (c *monitoring) retryCreate(
 	return true
 }
 
+func (c *monitoring) getDesiredGrafanaImage(cluster *corev1.StorageCluster) string {
```

Comment:
We will have to make changes to [manifest.go](https://github.com/libopenstorage/operator/blob/master/drivers/storage/portworx/manifest/manifest.go) and [portworx.go](https://github.com/libopenstorage/operator/blob/master/drivers/storage/portworx/portworx.go#L560) to get the grafana image from version manifest and then set it in the `status.desiredImages`.